adhocracy-plus/templates/account/email/account_already_exists.en.email

```diff
@@ -0,0 +1,21 @@
+{% extends 'email_base.'|add:part_type %}
+{% load i18n %}
+
+{% block subject %}{% blocktranslate %}Account Already Exists{% endblocktranslate %}{% endblock %}
+
+{% block content %}
+<p>
+    {% blocktranslate %}You are receiving this e-mail because you or someone else tried to signup for an
+    account using e-mail address:{% endblocktranslate %}
+</p>
+<p>
+    {{ user.email }}
+</p>
+<p>
+    {% blocktranslate %}However, an account using that e-mail address already exists.  In case you have
+    forgotten about this, please use the password forgotten procedure to recover
+    your account:{% endblocktranslate %}
+</p>
+{% block cta_url %}{{ password_reset_url }}{% endblock %}
```

Comment:
The cta block is not part of the content block, it renders incorrectly in the email. If you move the endblock from line 21 to here it works fine :)